iiod/queue.h

```diff
@@ -81,494 +81,540 @@
 /*
  * List definitions.
  */
-#define	LIST_HEAD(name, type)						\
-struct name {								\
-	struct type *lh_first;	/* first element */			\
-}
-
-#define	LIST_HEAD_INITIALIZER(head)					\
-	{ NULL }
-
-#define	LIST_ENTRY(type)						\
-struct {								\
-	struct type *le_next;	/* next element */			\
-	struct type **le_prev;	/* address of previous next element */	\
-}
+#define LIST_HEAD(name, type)                                                  \
```

Comment:
macros seems to get spaces vs tags before the backslash;
Linux style usually preferred tabs;
i guess either works if we are consistent